requirements.txt

```diff
@@ -0,0 +1,27 @@
+absl-py==0.3.0
+asn1crypto==0.24.0
+astor==0.7.1
+certifi==2018.8.13
+cffi==1.11.5
+colorama==0.3.7
+cryptography==2.3
+gast==0.2.0
+grpcio==1.14.1
+idna==2.7
+jmespath==0.9.3
+jwt==0.5.4
+Markdown==2.6.11
+mypy==0.620
+numpy==1.14.5
+protobuf==3.6.1
+pyasn1==0.4.2
+pycparser==2.18
+rsa==3.4.2
+s3transfer==0.1.13
+six==1.11.0
+tensorboard==1.10.0
+tensorflow==1.10.0
```

Comment:
concerned that this will break when users want to use tensorflow-gpu instead of tensorflow.  in the `setup.py` I'd made both of those optional and assumed that the user had installed one of them manually with pip before installing tf-encrypted.  I'd suggest we keep the requirements.txt file similar and try to manually specify our tensorflow version for circleci